src/main/java/seedu/duke/util/Parser.java

```diff
@@ -248,12 +250,38 @@ private Command prepareListNote(String userMessage) {
 
     private Command prepareListEvent(String userMessage) {
         return new ListEventCommand();
-    }
+    }*/
 
     private Command prepareViewNote(String userMessage) {
```

Comment:
Should we throw the SystemException here?
```suggestion
    private Command prepareViewNote(String userMessage) throws SystemException {
```

Reply:
Prepare functions should throw SystemException if necessary.